arch/lkl/kernel/vmlinux.lds.S

```diff
@@ -1,3 +1,4 @@
+#define RO_AFTER_INIT_DATA
```

Comment:
Is this an issue only on mingw? If so, we could declare RO_AFTER_INIT_DATA in arch/lkl/include/asm/vmlinux.lds.h so that it takes effect only for mingw.